client.go

```diff
@@ -463,6 +520,15 @@ func (s *SequentialCmd) Error() error {
 	return s.err
 }
 
+// test pub/sub
+func (s *SequentialCmd) WaitingPubAck(id string) (any, error) {
```

Comment:
Adding this check is to return the error from the previous method in the chain.
```
if s.err != nil {
		return nil, s.err
	}
```
It should not be discarded.

Reply:
ok. i will add it.